VideoSort.py

```diff
@@ -1188,7 +1188,7 @@ def construct_path(filename):
     if verbose:
         print('destination path: %s' % new_path)
 
-    if filename.upper() == new_path.upper():
+    if filename == new_path:
```

Comment:
This will cause failures on case insensitive filesystems, e. g. Windows and Mac.

Reply:
Because I can't merge a half of PR it's better to create separate PRs for separate issues. Thanks.

Reply:
Apologies, this wasn't supposed to be a part of the pull request.

I'll create a new one.